src/components/aeDivider/aeDivider.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <div :class="cssClass">
+  </div>
+</template>
+<script src='./aeDivider.js'/>
```

Comment:
Is it difficult to use only one of `"` and `'` for quoting of a value of tag attribute?